docs-core/src/main/java/com/sismics/docs/core/service/InboxService.java

```diff
@@ -235,6 +264,33 @@ private void importMessage(Message message) throws Exception {
             FileUtil.createFile(fileContent.getName(), null, fileContent.getFile(), fileContent.getSize(),
                     document.getLanguage(), "admin", document.getId());
         }
+
+        if (ConfigUtil.getConfigBooleanValue(ConfigType.INBOX_DELETE_IMPORTED)) {
+            message.setFlag(Flags.Flag.DELETED, true);
+        }
+    }
+
+    /**
+     * Fetches a HashMap with all tag names in lowercase as keys and their respective ids as values.
```

Comment:
Tag names are not lowercased, they are put as is in the hashmap.

Reply:
I originally planned for the matching not to be case-sensitive, because that makes it easier, but decided against it, because the rest is also case-sensitive. But maybe everything should be case-insensitive in a future update?
```suggestion
     * Fetches a HashMap with all tag names as keys and their respective ids as values.
```